pr-data.csv

```diff
@@ -4581,13 +4581,17 @@ https://github.com/intel/jndn-utils,b4f2fb28e48d8a4bc18410ec6c932c0c229f8884,.,c
 https://github.com/intel/jndn-utils,b4f2fb28e48d8a4bc18410ec6c932c0c229f8884,.,com.intel.jndn.utils.client.impl.DefaultSegmentedClientTest.verifyThatSegmentsAreRetrievedOnlyOnce,ID,Accepted,https://github.com/intel/jndn-utils/pull/8,
 https://github.com/intuit/karate,09bc49eb6fcfc029a09dbdb385137ff71985c89b,karate-core,com.intuit.karate.JsonUtilsTest.testPojoConversion,ID,MovedOrRenamed,,https://github.com/karatelabs/karate
 https://github.com/intuit/karate,2ca51aca126d691d87c09f3b476cbbaef58881ca,karate-core,com.intuit.karate.ScriptTest.testKarateToJson,ID,Deleted,,https://github.com/karatelabs/karate/commit/f40112e33891b11ba6188d5677053aed77136b04
+https://github.com/intuit/wasabi,eaac73406a80ca75fb9423bdf3faada67f955083,modules/analytics-objects,com.intuit.wasabi.analyticsobjects.EventTest.testHashCode,ID,Opened,https://github.com/intuit/wasabi/pull/457,
```

Comment:
I assume the ordering issue may be between these two rows.

Reply:
Okay, I was wrong. :) Please squash the 6 commits into 1.

Reply:
Have fixed the issue. However, I am unable to squash commits. I believe that is only available while merging through Github UI. Please let me know if anything else is needed to merge this PR.

Reply:
For squashing, see for example https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together (Of course, you'd need to `git push --force` after that.) Make sure to *not* close the PR while squashing.

Reply:
I closed this PR as it was created from the main branch of my forked repo and force push isn't possible in the main branch. Created a new branch and a pull request from it - https://github.com/TestingResearchIllinois/idoft/pull/1373